### PR TITLE
TST: added a CI test for operations

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Added a CI test for python 3.6, to support current operational environments.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Added a CI test.

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -27,7 +30,7 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Install NEP29 dependencies
+    - name: Install NEP29/Operational dependencies
       if: ${{ matrix.numpy_ver != 'latest'}}
       run: |
         pip install --no-binary :numpy: numpy==${{ matrix.numpy_ver }}
```
Comment (jklenzing) [inline .github/workflows/main.yml:36]:
```suggestion
        pip install --no-binary :numpy: numpy==${{ matrix.numpy_ver }}
        pip install netCDF<1.6.2
```

**Comment by jklenzing** [comment]
Still crashing.  Looks like it's installing an ancient version of xarray (0.16.2) for some reason (could be numpy or python related). all the failures have to do with the xarray interface.

**Comment by rstoneback** [comment]
xarray 0.16.2 is the last version that supports python 3.6 .

**Comment by rstoneback** [comment]
pandas didn't like getting a datarray input on line 1354 in utils.io.py . Updated from `data['time']` -> `data['time'].values` to provide an numpy array input instead.

**Comment by jklenzing** [comment]
Looking through the old logs, it looks like the netCDF4 install was working without the cap, it just took longer to fail and update versions.  Previously holding at 49 errors, watching the new tests to see how much the datetime compliance fits.

Also noting there are permission errors for the tempfile directory.

**Comment by rstoneback** [comment]
> Looking through the old logs, it looks like the netCDF4 install was working without the cap, it just took longer to fail and update versions. Previously holding at 49 errors, watching the new tests to see how much the datetime compliance fits.
> 
> Also noting there are permission errors for the tempfile directory.

Good news is the .values fix also sorted out the permission errors and 'slt' is not a dimension issues as well.

**Comment by jklenzing** [comment]
From digging through the various logs, it looks like we can drop both the pandas and netcdf limits for the operational cases

**Comment by aburrell** [comment]
> From digging through the various logs, it looks like we can drop both the pandas and netcdf limits for the operational cases

Maybe, but given that pip failed there earlier...  maybe we try removing them later?

**Comment by jklenzing** [comment]
> > From digging through the various logs, it looks like we can drop both the pandas and netcdf limits for the operational cases
> 
> Maybe, but given that pip failed there earlier... maybe we try removing them later?

I'll try a separate test branch.

**Review by rstoneback — APPROVED** [review]

**Comment by rstoneback** [comment]
Started a pysat RC https://github.com/pysat/pysat/pull/1068 pending incorporation of this pull.

**Comment by aburrell** [comment]
Rebased into the RC to simplify the release process.

**Pull request merged**